app/resources/en/translation/executors/address.json

```diff
@@ -35,12 +35,12 @@
     },
     "newPostCode": {
       "required": {
-        "summary": "Enter a postcode",
+        "summary": "You haven&rsquo;t entered the postcode",
```

Comment:
Please revert this to the original text. 
From UX testing and GDS we should not use the pronoun "You" at the start of a sentence